backend/src/models/user/postgres.js

```diff
@@ -66,6 +66,32 @@ const UserRepo = (postgres) => {
     }
   };
 
+  /**
+   * Retrieve the user when the email is given
+   * 
+   * @type {string}
+   */
+  const getUserInfoByEmailSQL = `
+    SELECT * FROM users WHERE email=$1;`;
+
+  /**
+   * Uses getUserInfoByEmailSQL to retrieve the user, and return either (user, null), or (null, error)
+   * 
+   * @param  {string} email - Email of the user to retrieve
+   * @return {Array<{0: User, 1: String}>} - Array with Rafflebay User Object and error (only one or the other)
+   */
+  const getUserInfoByEmail = async (email) => {
+    const values = [email];
+    try {
+      const client = await postgres.connect();
+      const res = await client.query(getUserInfoByEmailSQL, values);
+      client.release();
+      return [res.rows[0], null];
+    } catch (err) {
+      return [null, err];
+    }
+  };
```

Comment:
Was this just a copy paste from below?

Reply:
Yea it should have been a cut and paste. Just because I don't know how build time dependencies work in JS